### PR TITLE
Selected Locations Use Case

### PR DESCRIPTION
**Pull request by seanwoo12**
added implementation of selected locations use case, fixed some bugs

### CODE REVIEW

File: src/main/java/app/MainWithDB.java
Comment (teddyyang8) [inline src/main/java/app/MainWithDB.java:1]: why is suggestedlocationviewmodel passed into selectedlocationview?

File: src/main/java/use_case/selected_locations/SelectedLocationsInteractor.java
Comment (teddyyang8) [inline src/main/java/use_case/selected_locations/SelectedLocationsInteractor.java:1]: line 32 return?

Comment (teddyyang8) [inline src/main/java/use_case/selected_locations/SelectedLocationsInteractor.java:1]: the map isnt being used? shouldnt it be final SelectedLocationsOutputData selectedLocationsOutputData =
                new SelectedLocationsOutputData(Map);?

**Review by teddyyang8 — CHANGES_REQUESTED** [review]

File: src/main/java/view/SelectedLocationView.java
Comment (teddyyang8) [inline src/main/java/view/SelectedLocationView.java:1]: since the viewmodel is not being used here

**Review by teddyyang8 — CHANGES_REQUESTED** [review]

**Review by teddyyang8 — APPROVED** [review]

**Review by shirleyyzhang — APPROVED** [review]

**Pull request merged**